eva_sub_cli/validators/validator.py

```diff
@@ -148,25 +152,14 @@ def check_if_file_missing(self):
         return files_missing, missing_files_list
 
     def update_config_with_validation_result(self):
-        self.sub_config.set(VALIDATION_RESULTS, value=self.results)
+        self.sub_config.set(VALIDATION_RESULTS_KEY, value=self.results)
```

Comment:
I didn't realise we were already storing the validation results in the config... should we remove it from the config given that it's in a separate file now?

Reply:
That's a good point. @tcezard any issues if I remove the the validations from the config file.

Reply:
Probably not. I don't think it is used anywhere and it makes more sense to have it in the separate file.

Reply:
I think we should remove this function and make sure the `READY_FOR_SUBMISSION_TO_EVA` key is set in `_assess_validation_results `

Reply:
done